Include/internal/pycore_code.h

```diff
@@ -0,0 +1,27 @@
+#ifndef Py_INTERNAL_CODE_H
+#define Py_INTERNAL_CODE_H
+#ifdef __cplusplus
+extern "C" {
+#endif
+ 
+typedef struct {
+    PyObject *ptr;  /* Cached pointer (stealed reference) */
```

Comment:
stolen reference?  I'm more used to seeing that around something that can be called ... do you just mean that the object pointed to SHOULD not get an incref for this?  (And is that to avoid circular references?)

Reply:
Ah, I was wrong.  It is borrowed reference, not stolen.